INSTALL.md

```diff
@@ -107,7 +107,7 @@ itself: currently F* binary packages for 2021.x are built with OCaml
    the following command to install the packages required to compile
    OCaml programs extracted from F\* code:
 
-       $ opam install ocamlfind batteries stdint zarith ppx_deriving_yojson pprint ppxlib=0.22.0 ocaml-compiler-libs
+       $ opam install ocamlfind batteries stdint zarith ppx_deriving_yojson pprint ppxlib ocaml-compiler-libs
```

Comment:
I think also should also change this location https://github.com/FStarLang/FStar/blob/7dea970b4f3eb51a4881e5cb72e8d478d6f85797/fstar.opam#L19

Reply:
I think the .opam file should be ok still as-is since it only requires ppxlib at least 0.22.0 and ocaml at least 4.07.0. As a result, opam should be able to figure out versions of ppxlib and ocaml that don't conflict. This could be versions 0.22.0 and 4.07.0, or versions 0.26.0 and 4.14.x. 

The main problem with INSTALL.md is that it tells opam it MUST install ppxlib version 0.22.0. But looking at it again, it should probably just provide the same ppxlib constraint, that is `'ppxlib>=0.22.0'`. I'll update INSTALL.md to do that.

Reply:
Make sense. Thanks for clarification